packages/form-js-editor/package.json

```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "all": "run-s lint test build",
     "build": "run-p bundle generate-types",
-    "bundle": "rollup -c --failAfterWarnings",
```

Comment:
Ah I forgot about this one, flatpickr actually throws out warnings so I need to consider something for this